origin-discovery/package.json

```diff
@@ -13,9 +13,12 @@
   },
   "dependencies": {
     "apollo-server": "^2.1.0",
+    "apollo-server-express": "^2.1.0",
```

Comment:
can the apollo-server dependency above get removed since looks like apollo-server-express replaces it ?

Reply:
Good catch!